ui/packages/ui/src/app/pages/connectors/ConnectorTask.tsx

```diff
@@ -1,15 +1,18 @@
-import { Label, Split, SplitItem } from '@patternfly/react-core';
+import { Button, Flex, FlexItem, Label, Popover } from '@patternfly/react-core';
 import * as React from "react";
-import { HelpInfoIcon } from 'src/app/components/formHelpers/HelpInfoIcon';
 import { ConnectorState } from "src/app/shared";
 import "./ConnectorTask.css";
-
 export interface IConnectorTaskProps {
   status: string;
+  connName: string;
   taskId: string;
   errors?: any;
   i18nTask: string;
+  i18nRestart: string;
   i18nTaskStatusDetail: string;
+  i18nTaskErrorTitle: string;
+  setConnectorToRestart: (connName: string) => void;
+  showRestartConfirmationDialog: () => void;
```

Comment:
There will need to be a separate confirmation which indicates restart of connector name, task id